R/build-home-authors.R

```diff
@@ -76,9 +76,19 @@ data <- data_authors(pkg, roles)
   authors <- data$main %>% purrr::map_chr(author_desc, comment = FALSE)
 
   bullets <- c(
-      markdown_text2(pkg$meta$authors$sidebar$before),
+      markdown_text2(
+        pkg$meta$authors$sidebar$before,
+        pkg = pkg,
+        what = pkgdown_field(pkg = pkg, "authors", "sidebar", "before"),
```

Comment:
I'm not especially happy with how cumbersome this has become. `what` is only needed when `inline` is `TRUE` (as when it is `FALSE`, anything will end up in a paragraph).

Reply:
Since `markdown_text2()` already has `pkg`, why not change `what` to `where` and do:

```R
markdown_text2(pkg$meta$authors$sidebar$before, pkg, 
  where = c("authors", "sidebar", "before"),
  inline = TRUE
))
```

Given that you only need where if `inline` is `TRUE`, it might be better to have a separate function:

```R
markdown_inline(pkg$meta$authors$sidebar$before, pkg,  where = c("authors", "sidebar", "before"))
```

The other variant would be called `markdown_block()`